doc/flame/effects.md

```diff
@@ -350,6 +350,8 @@ EffectController({
     int? repeatCount,
     bool infinite = false,
     double startDelay = 0.0,
+    VoidCallback? onPeak,
+    VoidCallback? onReversePeak,
```

Comment:
The "reversePeak" term seem kinda confusing. Maybe we should call these callbacks `onMax/onMin`, or `onReachMax/onReacMin`, or `onArriveAtMax/onArriveAtMin`, etc -- but to remain in concord with the existing terminology of `atMaxDuration/atMinDuration`.

Reply:
`onMin` and `onMax` I think is good, with some good docs of course.

Reply:
Here the method names are still old.